src/Surfnet/Stepup/DateTime/DateTime.php

```diff
@@ -165,4 +166,9 @@ public function __toString()
     {
         return $this->format(self::FORMAT);
     }
+
+    public function jsonSerialize()
+    {
+        return $this->format(self::FORMAT);
```

Comment:
Question: Why make this `JsonSerializable` rather than the `AuditLogEntry`? Also, since the format is really important, please add a test to ensure that when the format or serialisation or ... changes, it is reflected in the test.